README.md

```diff
@@ -38,9 +44,15 @@ from transformers import AutoTokenizer
 import soundfile as sf
 import torch
 
-device = "cuda:0" if torch.cuda.is_available() else "cpu"
+device = "cpu"
+if torch.cuda.is_available():
+    device = "cuda:0"
+if torch.backends.mps.is_available():
+    device = "mps"
+if torch.backends.xpu.is_available():
```

Comment:
Should we keep it minimal and just use cuda/mps/cpu?

```suggestion
device = "cuda:0" if torch.cuda.is_available() else "mps" if torch.backends.mps.is_available() else  "cpu"
```

Reply:
xpu is used in my group by members that use ARC 770 systems.